cbr/cbr.go

```diff
@@ -0,0 +1,289 @@
+// Package cbr ...
+package cbr
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/contextbasedrestrictionsv1"
+)
+
+const (
+	kubernetes_service = "containers-kubernetes"
+	is_service         = "is"
+	cos_service        = "cloud-object-storage"
+	keyProtect_service = "kms"
+)
+
+type CBR struct {
+	VPC        []string `json:"VPC,omitempty"`        //The VPCs allowed in the zone. Input in the form 'value,value,...' where value is VPC CRN
+	Address    []string `json:"Address,omitempty"`    //The list of addresses in the zone. Only addresses of type 'ipAddress', 'ipRange', and 'subnet' are allowed in a comma delimited format.
+	ServiceRef []string `json:"ServiceRef,omitempty"` //The service refs in the zone. Input in the form 'value,value,...'
+}
+
+// CBRInterface ...
+type CBRInterface interface {
+
+	// CreateCBRZone ...
+	CreateCBRZone(name string, cbrInput CBR) (string, error)
```

Comment:
Will the VPC id be mandatory for creating CBR zone?

Reply:
@Bhagyashreek8 ideally either vpc or address or service-ref needs to be given else there is no sense of creating a zone. The API or CLI will error out we need to provide one of these.